src/main/java/com/avispl/symphony/dal/communicator/lg/lcd/LgLCDDevice.java

```diff
@@ -171,64 +622,460 @@ public List<Statistics> getMultipleStatistics() throws Exception {
+		checkControlPropertyBeforeAddNewProperty(controlVolume, advancedControllableProperties);
+
+		String[] inputDropdown = cacheMapOfPriorityInputAndValue.values().stream().sorted().collect(Collectors.toList()).toArray(new String[0]);
+		AdvancedControllableProperty controlInputSource = controlDropdown(statistics, inputDropdown, inputGroupName + LgLCDConstants.INPUT_SELECT,
+				cacheMapOfPropertyNameAndValue.get(LgLCDConstants.INPUT_SELECT));
+		checkControlPropertyBeforeAddNewProperty(controlInputSource, advancedControllableProperties);
+
+		String inputTypeValue = isInputTypePC ? LgLCDConstants.PC : LgLCDConstants.DTV;
+		AdvancedControllableProperty controlInputType = controlDropdown(statistics, LgLCDConstants.INPUT_TYPE_DROPDOWN, inputGroupName + LgLCDConstants.INPUT_TYPE, inputTypeValue);
+		checkControlPropertyBeforeAddNewProperty(controlInputType, advancedControllableProperties);
+
+		String[] pmdDropdown = EnumTypeHandler.getEnumNames(PowerManagement.class);
+		AdvancedControllableProperty controlPMD = controlDropdown(statistics, pmdDropdown, powerManagementGroupName + LgLCDConstants.STAND_BY_MODE,
+				cacheMapOfPropertyNameAndValue.get(LgLCDConstants.STAND_BY_MODE));
+		checkControlPropertyBeforeAddNewProperty(controlPMD, advancedControllableProperties);
+
+		String pmdModeValue = retrieveDataByCommandName(commandNames.PMD_MODE, commandNames.PMD_MODE_PARAM);
+		String[] pmdModeDropdown = EnumTypeHandler.getEnumNames(PowerManagementModeEnum.class);
+		String pmdValue = EnumTypeHandler.getNameEnumByValue(PowerManagementModeEnum.class, pmdModeValue);
+		AdvancedControllableProperty controlPMDMode = controlDropdown(statistics, pmdModeDropdown, powerManagementGroupName + LgLCDConstants.PM_MODE, pmdValue);
+		checkControlPropertyBeforeAddNewProperty(controlPMDMode, advancedControllableProperties);
+	}
+
+	/**
+	 * Retrieve data by command name
+	 *
+	 * @param command the command is command to send the request get the data
+	 * @param param the param is param to send the request get the data
+	 * @return String is data response from the device or None if response fail
+	 */
+	private String retrieveDataByCommandName(commandNames command, commandNames param) {
+		try {
+			byte[] response = send(LgLCDUtils.buildSendString((byte) monitorID, LgLCDConstants.commands.get(command), LgLCDConstants.commands.get(param)));
+			return digestResponse(response, command).toString();
+		} catch (Exception ex) {
+			logger.debug("Error while retrieve monitoring data by name: " + command);
+		}
+		failedMonitor.add(command.name());
+		return LgLCDConstants.NONE;
 	}
 
 	/**
 	 * This method is used to get the current display power status
 	 *
 	 * @return powerStatus This returns the calculated xor checksum.
+	 * @throws SocketTimeoutException if the connection is reset many times
 	 */
-	private LgLCDConstants.powerStatusNames getPower() {
+	private LgLCDConstants.powerStatusNames getPower() throws Exception {
```

Comment:
We handled the exception in method body so there is no need to throw an exception here